### DISCUSSION TITLE
How to send "value" in analytics event

### DISCUSSION THREAD

**Discussion by antondanilov95** [invertase/react-native-firebase#5174] [Q&A]
Please, can you help me with sending of value in the event. It's not showing in firebase console. How to send it correctly?

I tried like this but it doesn't work
```
Analytics().logEvent('ecommerce_purchase', {
      value: 456,
      orderNumber: '1234',
    });
```

"@react-native-firebase/analytics": "^6.2.0",
"@react-native-firebase/app": "^6.2.0",
<img width="698" alt="Screenshot 2021-04-14 at 23 25 27" src="https://user-images.githubusercontent.com/23694724/114774563-fceda080-9d78-11eb-95da-0c935b006fad.png">

**Comment by mikehardy** [comment]
Hi there!
Your versions are very very old, you should upgrade to current.
Maybe relevant? https://github.com/invertase/react-native-firebase/issues/4578
